README.md

```diff
@@ -33,6 +33,13 @@ echo "https://api.cloudflare.com/client/v4/pages/webhooks/deploy_hooks/ENDPOINT_
 
 you can now use the `rebuild` script in the root of the repository to initiate manual deployments without having to push changes to `main`.
 
+7. To run tests, create a new file called .env.test in the backend folder. Set the following values:
```

Comment:
note that .env.test isn't in your .gitignore so this is already present in the repo, meaning it doesn't have to be created by the user. which is probably a good thing.